cluster/charts/crossplane/README.md

```diff
@@ -87,6 +87,7 @@ and their default values.
 | `securityContextCrossplane.runAsGroup` | Run as group for Crossplane | `65532` |
 | `securityContextCrossplane.allowPrivilegeEscalation` | Allow privilege escalation for Crossplane | `false` |
 | `securityContextCrossplane.readOnlyRootFilesystem` | ReadOnly root filesystem for Crossplane | `true` |
+| `serviceAccount.crossplane.create` | Setting to `false` means the crossplane ServiceAccount is managed outside this chart | `true` |
```

Comment:
Instead of an on-off switch, can we accept the name of the `ServiceAccount` you'd like to use? Crossplane could create its own if no name is given.

Reply:
That will not address my specific use case. I want to create it outside of the helm chart completely because it needs to be created with AWS arn specific annotations for rbac purposes.

I suppose if there was a way to add annotations to the service account that would help, but as we already have the code for creating the service account I'd prefer to just switch it off completely in this helm chart. I accept that this is not my codebase, so others may not agree.

Reply:
What I meant was that if you provide the name of the ServiceAccount you created, the chart would make Crossplane use it - with whatever anootations/labels you deem appropriate for that ServiceAccount. If a name isn't provided, only then it'd create its own. Wouldn't that work for you?

Reply:
@jeremyspykerman, with the merge of #2788 your usecase is handled. In release 1.6.0 the helm charts allows you to set annotation for the service account, to be used for AWS IRSA stuff.

Reply:
I agree that what you've proposed is a better solution @muvaf . I just haven't had time to refactor to take this approach.

Also now that the option mentioned by @dwerder is available, I'll have to reconsider the approach we're using. It could be quite some time, but once we make a decision regarding which approach we want to take I'll either rework the PR, or adapt our code to use the annotation features from #2788

Reply:
chiming in here RE https://crossplane.slack.com/archives/CEG3T90A1/p1643853900373079 on slack...

i would still like to see the option to pass a custom SA name. reason being i use `eksctl` to manage well-known service accounts, like `ecr-access`. so i would love to re-use the existing account/trust-relationship it manages for me.

otherwise, i need to add additional IAM automation to set up the trust relationship for the `crossplane` SA.